messaging/counter/tasks/balances.ts

```diff
@@ -0,0 +1,96 @@
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import * as dotenv from "dotenv";
+import { ethers } from "ethers";
+import { getAddress } from "@zetachain/addresses";
+import ZetaEth from "@zetachain/interfaces/abi/json/contracts/Zeta.eth.sol/ZetaEth.json";
+
+const walletError = `
+❌ Error: Wallet address not found.
+
+To resolve this issue, please follow these steps:
+
+* Set your PRIVATE_KEY environment variable. You can write
+  it to a .env file in the root of your project like this:
+
+  PRIVATE_KEY=123... (without the 0x prefix)
+  
+  Or you can generate a new private key by running:
+
+  npx hardhat account --save
+
+* Alternatively, you can fetch the balance of any address
+  by using the --address flag:
+  
+  npx hardhat balances --address <wallet_address>
+`;
+
+async function fetchNativeBalance(
+  address: string,
+  provider: ethers.providers.JsonRpcProvider
+) {
+  const balance = await provider.getBalance(address);
+  return parseFloat(ethers.utils.formatEther(balance)).toFixed(2);
+}
+
+async function fetchZetaBalance(
+  address: string,
+  provider: ethers.providers.JsonRpcProvider,
+  networkName: string
+) {
+  if (networkName === "athens") return "";
+  const zetaAddress = getAddress({
+    address: "zetaToken",
+    networkName,
+    zetaNetwork: "athens",
+  });
+  const contract = new ethers.Contract(zetaAddress, ZetaEth, provider);
+  const balance = await contract.balanceOf(address);
+  return parseFloat(ethers.utils.formatEther(balance)).toFixed(2);
+}
+
+async function fetchBalances(
```

Comment:
move to arrow function

Reply:
This will be imported from the toolkit (or "tasks"), so I've created an issue there.

https://github.com/zeta-chain/toolkit/issues/4